plugins/uploader/download.sh

```diff
@@ -63,13 +63,14 @@ fi
 
 url=$host/v0/organizations/$organization/clusters/$cluster/data/$datatype/config
 set +x
+status=0
 results=$(curl -X POST $url \
   -L \
   -H "Authorization: Bearer ${FAIRWINDS_TOKEN//[$'\t\r\n']}" \
   -H "Accept: application/x-yaml" \
   -O $file \
-  --fail 2>&1)
-if [ $? -ne 0 ]
+  --fail 2>&1) || status=$?
```

Comment:
why not just remove `--fail`?

Reply:
So that I can still fail for a 500 or 400 or other status code. It feels like that's still the appropriate choice. I could change course on that though.

Reply:
👍 makes sense